src/components/layout/layout.tsx

```diff
@@ -318,6 +319,7 @@ const Layout = ({
             )}
 
             {!no_live_chat && <LiveChat is_banner_shown={show_cookie_banner} />}
+            <WhatsApp />
```

Comment:
Seems it should be after `{FooterNav}`. Isn't it?
Is icon should hide right now?

Reply:
Hi @NikitK-deriv why it should be after {FooterNav} ? 🤔 

No we won’t hide it , we showing it to Nigeria and South African clients 🙏

Reply:
Ohh okay then 👍  I passed it.